src/helpers/config/types.ts

```diff
@@ -36,9 +36,6 @@ export type NetworkConfig = {
   baseAsset: string;
   baseAssetWrappedAddress?: string;
   rewardTokenSymbol: string;
-  rewardTokenAddress: string;
-  rewardTokenDecimals: number;
-  incentivePrecision: number;
   usdMarket?: boolean;
```

Comment:
why is this still needed @sendra / @aschmidt20 ?

Reply:
These are removed in #199 now that we get the reward token symbol from the new incentives provider